scispacy/file_cache.py

```diff
@@ -96,9 +97,14 @@ def filename_to_url(filename: str, cache_dir: Optional[str] = None) -> Tuple[str
 
 def http_get(url: str, temp_file: IO) -> None:
     req = requests.get(url, stream=True)
+    total = int(req.headers.get("content-length", 0))
+    pbar = tqdm(total=total, unit="iB", unit_scale=True, unit_divisor=1024)
     for chunk in req.iter_content(chunk_size=1024):
         if chunk:  # filter out keep-alive new chunks
             temp_file.write(chunk)
```

Comment:
I think you forgot to delete this line on this new pr. I think that is what is causing the test failures.